cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceStateMachineTest.java

```diff
@@ -257,6 +269,15 @@ public void testNewToProposalReadyRemoveBroker(Vertx vertx, VertxTestContext con
         this.krNewToProposalReady(vertx, context, CruiseControlEndpoints.REMOVE_BROKER, kcRebalance);
     }
 
+    /**
+     * See the {@link KafkaRebalanceStateMachineTest#testNewToProposalReadyRebalance} for description
```

Comment:
wrong javadoc? rebalance is not related to it.
the same applies to all the other following tests.